sensor_msgs/Range[5] ir_front_left
sensor_msgs/Range[5] ir_front_right
sensor_msgs/Range[5] ir_back_left
sensor_msgs/Range[5] ir_back_right
#sensor_msgs/Range[5] ir_back
#sensor_msgs/Range[5] ir_us_left
#sensor_msgs/Range[5] ir_us_right
